build-system/babel-plugins/babel-plugin-transform-jss/index.js

```diff
@@ -86,12 +164,43 @@ module.exports = function ({template, types: t}) {
       },
 
       CallExpression(path, state) {
+        const callee = path.get('callee');
+
+        // Replace users that import JSS.
+        const importDeclaration = findImportDeclaration(callee, 'useStyles');
+        if (importDeclaration) {
+          const {parentPath} = path;
+          // Such as: useStyles().myClassName
+          if (replaceMemberExpression(importDeclaration, path, parentPath)) {
+            return;
+          }
+          // Such as: const classes = useStyles();
+          if (parentPath.isVariableDeclarator()) {
+            const {referencePaths} = path.scope.bindings[
+              parentPath.node.id.name
+            ];
+            const replacedReferenceCount = referencePaths.reduce(
+              (count, {parentPath}) =>
+                replaceMemberExpression(importDeclaration, path, parentPath)
+                  ? count + 1
+                  : count,
+              0
+            );
+            if (referencePaths.length === replacedReferenceCount) {
+              parentPath.remove();
+            }
+            return;
+          }
+          // Or unknown use, deopt.
+          return;
+        }
+
+        // Replace JSS exporter
         const {filename} = state.file.opts;
         if (!isJssFile(filename)) {
           return;
         }
 
-        const callee = path.get('callee');
         if (!callee.isIdentifier({name: 'createUseStyles'})) {
```

Comment:
Nit:
```suggestion
        if (!callee.referencesImport('react-jss', 'createUseStyles')) {
```

Reply:
Previous tests aren't passing when doing this for some reason. It's a head-scratcher.